database/migrations/2024_07_12_095213_fix_missing_sqlite_foreign_keys.php

```diff
@@ -32,18 +32,11 @@ public function up(): void
                 "updated_at" datetime, 
                 "user_id" integer not null, 
                 "memo" text, 
-                "r_servers" integer not null default \'0\', 
-                "r_nodes" integer not null default \'0\', 
-                "r_allocations" integer not null default \'0\', 
-                "r_users" integer not null default \'0\', 
-                "r_eggs" integer not null default \'0\', 
-                "r_database_hosts" integer not null default \'0\', 
-                "r_server_databases" integer not null default \'0\', 
                 "identifier" varchar, 
                 "key_type" integer not null default \'0\', 
                 "last_used_at" datetime, 
                 "expires_at" datetime,
-                "r_mounts" integer not null default \'0\',  
+                "permissions" text,  
```

Comment:
This migration already exists, so this should be put in a new migration, right?

Reply:
Yeah, that's because this PR is so old.